spin.lic

```diff
@@ -13,16 +13,18 @@ class Spin
   def initialize
     arg_definitions = [
       [
-        { name: 'material', options: %w[burlap silk linen], description: 'fiber to spin' },
+        { name: 'material', options: %w[silk cotton linen wool burlap], description: 'fiber to spin' },
         { name: 'weight', options: %w[fine thin average thick heavy yarn], optional: true, description: 'Thickness, defaults to heavy' }
       ]
     ]
     args = parse_args(arg_definitions)
 
     mat_data = {
       'silk' => { name: 'silk', index: 1, cost: 187, count: 13 },
```

Comment:
This data looks like it might belong in `base-crafting`. Food for thought in the future.